app/controllers/users_controller.rb

```diff
@@ -12,7 +12,9 @@ def index
       mentoring:  'Helping as a Mentor',
       organizing: 'Helping as an Organizer'
     }
-    @users = User.ordered(params[:sort], params[:direction]).group('users.id').with_all_associations_joined
+    @users = User.ordered(params[:sort],
+                          params[:direction]).group('users.id').with_all_associations_joined.
```

Comment:
This is the way I understood it from the gem's instructions. They called the `.per()` a scope as well. And they put it chained to the `ordered` method. 
I figured this was the safer way to go. 
16 is mostly arbitrary; my front end guru will decide on a default value for the whole site.:-)